src/constructors/trait.js

```diff
@@ -1,8 +1,15 @@
+// @flow
 import concat from './concat'
 
-export default (name, definitions, cb) => {
+type Definitions = {
+  [category: string]: {
+    [option: string]: string
+  }
+}
+
+export default (name: string, definitions: Definitions, cb: Function): Function => {
```

Comment:
what are the parameters and return value of `cb`? I found something in the code like `cb(obj)` but I have no idea what `obj` is... soooo `cb: (obj: Object) => void`